library/ecp.c

```diff
@@ -410,6 +410,12 @@ static const mbedtls_ecp_curve_info ecp_supported_curves[] =
 #endif
 #if defined(MBEDTLS_ECP_DP_SECP192K1_ENABLED)
     { MBEDTLS_ECP_DP_SECP192K1,    18,     192,    "secp192k1"         },
+#endif
+#if defined(MBEDTLS_ECP_DP_CURVE25519_ENABLED)
+    { MBEDTLS_ECP_DP_CURVE25519,   29,     256,    "x25519"            },
+#endif
+#if defined(MBEDTLS_ECP_DP_CURVE448_ENABLED)
+    { MBEDTLS_ECP_DP_CURVE448,     30,     448,    "x448"              },
```

Comment:
Like above, these curves are automatically offered by TLS in the handshake, but the Mbed TLS x448 implementation is not integrated completely yet, and the handshake will fail if this curve is selected by the peer. I think that we should remove this for now.

@wintersteiger I am sorry for the flip-flopping.